src/main/java/org/folio/consortia/service/impl/UserServiceImpl.java

```diff
@@ -89,7 +87,7 @@ public User prepareShadowUser(UUID userId, String tenantId) {
       if (Objects.nonNull(userOptional.getId())) {
         user.setId(userId.toString());
         user.setPatronGroup(PATRON_GROUP);
-        user.setUsername(userOptional.getUsername() + HelperUtils.randomString(RANDOM_STRING_COUNT));
+        user.setUsername("shadow-" + userOptional.getUsername());
```

Comment:
We will have some discussing regarding username on todays refinement, also here is related PR:
https://github.com/folio-org/mod-consortia/pull/76

Reply:
Changed thanks,